dsc-website-client/src/components/MansoryPost.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import TagRow from './tag-row'
+
+
+export default function MasonryPost ({post, tagsOnTop}) {
+
+    const windowWidth = window.innerWidth
+
+    const imageBackground = {backgroundImage:  `url("${require(`../images/${post.image}`)}")`};
+
+    const style = windowWidth > 800 ? {...imageBackground, ...post.style} : imageBackground
```

Comment:
Doesn't Material UI have a component to handle images when the screen shrinks?
Take a look at material UI Grids - https://material-ui.com/components/grid/
...
This way the image won't take up the width of the entire screen

Reply:
Not sure what you mean by this? 
So that there's white space instead of forming a full square if there's an uneven number of events?

Reply:
Here you're dealing with the width of the window. And based on that you modify the image no?
Material UI has Grid which is like a div but dynamically resized the divs when the window size shrinks.

Reply:
oh gotcha i'll look into it